src/components/action/action.tsx

```diff
@@ -60,6 +60,18 @@ export class Action implements InteractiveComponent, LoadableComponent {
    */
   @Prop({ reflect: true }) indicator = false;
 
+  /**
+   * Specifies the text label to display `indicator` is `true`.
+   */
+  @Prop() indicatorMessage = "";
+
+  /**
+   * Specifies the text label to display `indicator` is `true`.
+   *
+   * @default "Unread changes"
+   */
+  @Prop() intlIndicator?: string = TEXT.indicator;
```

Comment:
I think we were going to be removing the optional props with JSDoc references of "`?`" from the docs? 

And in this case if the prop isn't present there is no message read back to AT if line 196 remains the same.